rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -483,6 +483,19 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   std::map<std::string, std::vector<std::string>>
   get_service_names_and_types() const;
 
+  /// Return a map of existing service names to list of service types for a specific node.
```

Comment:
```suggestion
  /// Return a map of existing service names to a list of service types for a specific node.
```

Reply:
Currently the line matches the other docblock for similar function. Should I change those too?

Reply:
Hmm, reading it again I think the original comment is grammatically correct. I think we can disregard this.